pkgs/by-name/by/byedpi/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "byedpi";
+  version = "0.12";
+
+  src = fetchFromGitHub {
+    owner = "hufrea";
+    repo = "byedpi";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-LS4Dy2xTssYtN7EqWlGRh2MLTlf0kokJ2rHWuarokPE=";
+  };
+
+  installPhase = ''
+    install -Dm755 ciadpi $out/bin/ciadpi
+  '';
+
+  strictDeps = true;
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "SOCKS proxy server implementing some of DPI bypass methods";
```

Comment:
s/some of/some/

Thank you for your pull request! :wave:

Reply:
Thanks! But maybe completely rewrite description? Like "SOCKS5 proxy server that bypasses dpi-based censorship"?